docs/cudf/source/user_guide/performance_comparisons.ipynb

```diff
@@ -0,0 +1,1647 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    plt.show()
</pre><p>Does this plot add anything that the (single-entry) table does not?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/12595/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>